### ISSUE TITLE
ftfy does not currently fix mojibake from Windows-1254 (Turkish) to UTF-8

### ISSUE THREAD

**Issue by nth-attempt** [rspeer/python-ftfy#154]
Input: `ğŸ˜Š `
Desired result: `😊`

```
U+011F  ğ       [Ll] LATIN SMALL LETTER G WITH BREVE
U+0178  Ÿ       [Lu] LATIN CAPITAL LETTER Y WITH DIAERESIS
U+02DC  ˜       [Sk] SMALL TILDE
U+0160  Š       [Lu] LATIN CAPITAL LETTER S WITH CARON
```

**Comment by rspeer** [comment]
Updated the title to describe the particular encoding mixup that isn't being fixed here.

I worry that adding this support would slow down ftfy.

**Comment by rspeer** [comment]
Looking back, this is something I fixed in 6.0.
